### ISSUE TITLE
Dependency Error puppeteer-extra-plugin-adblocker

### ISSUE THREAD

**Issue by PaulPiatek** [berstend/puppeteer-extra#370]
There is a dependecy error on the latest build (maybe because of the old dependecy to @cliqz/adblocker-puppeteer

First install puppeter (ok):

```
npm install puppeteer  
added 54 packages, and audited 54 packages in 7s  
7 packages are looking for funding  
  run npm fund for details  
found 0 vulnerabilities
```

Then install puppeteer-extra-plugin-adblocker:

```
npm install puppeteer-extra-plugin-adblocker  
npm ERR! code ERESOLVE  
npm ERR! ERESOLVE unable to resolve dependency tree  
npm ERR!   
npm ERR! Found: @types/puppeteer@5.4.0  
npm ERR! node_modules/@types/puppeteer  
npm ERR!   @types/puppeteer@"*" from puppeteer-extra@3.1.15  
npm ERR!   node_modules/puppeteer-extra  
npm ERR!     peer puppeteer-extra@"*" from puppeteer-extra-plugin-adblocker@2.11.8  
npm ERR!     node_modules/puppeteer-extra-plugin-adblocker  
npm ERR!       puppeteer-extra-plugin-adblocker@"*" from the root project  
npm ERR!   
npm ERR! Could not resolve dependency:  
npm ERR! peer @types/puppeteer@"3.x" from @cliqz/adblocker-puppeteer@1.18.4  
npm ERR! node_modules/@cliqz/adblocker-puppeteer  
npm ERR!   @cliqz/adblocker-puppeteer@"^1.4.0" from puppeteer-extra-plugin-adblocker@2.11.8  
npm ERR!   node_modules/puppeteer-extra-plugin-adblocker  
npm ERR!     puppeteer-extra-plugin-adblocker@"*" from the root project  
npm ERR!   
npm ERR! Fix the upstream dependency conflict, or retry  
npm ERR! this command with --force, or --legacy-peer-deps  
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by Niek** [comment]
This will be fixed once https://github.com/cliqz-oss/adblocker/pull/1427 is merged.

**Comment by Niek** [comment]
Fixed now with release [1.18.5 of @cliqz/adblocker-puppeteer](https://www.npmjs.com/package/@cliqz/adblocker-puppeteer), please try to install again. Closing this issue.

**Comment by PaulPiatek** [comment]
Yes it works now. Thanks for your support.
